src/AspNetCoreModuleV2/AspNetCore/Inc/applicationinfo.h

```diff
@@ -14,6 +19,9 @@ HRESULT
     _Out_ IAPPLICATION       **pApplication
     );
 
+extern BOOL     g_fRecycleProcessCalled;
```

Comment:
We should figure out where we are externing global variables. Do we want to move all of them outside of the precomp.hxx file or not?

Reply:
We want to remove all of them ideally.

Reply:
what's the benefit?

Reply:
It's easy to track who uses and sets global variable when it's part of some class.